docs/kafka/message-browsing-kafka/README.adoc

```diff
@@ -0,0 +1,158 @@
+:getting-started-url-kafka: kafka/getting-started-kafka/README.adoc
+:kafka-bin-scripts-url-kafka: kafka/kafka-bin-scripts-kafka/README.adoc
+:kafkacat-url-kafka: kafka/kcat-kafka/README.adoc
+:quarkus-url-kafka: kafka/quarkus-kafka/README.adoc
+:nodejs-url-kafka: kafka/nodejs-kafka/README.adoc
+:rhoas-cli-getting-started-url-kafka: kafka/rhoas-cli-getting-started-kafka/README.adoc
+:topic-config-url-kafka: kafka/topic-configuration-kafka/README.adoc
+:consumer-config-url-kafka: kafka/consumer-configuration-kafka/README.adoc
+:access-mgmt-url-kafka: kafka/access-mgmt-kafka/README.adoc
+:metrics-monitoring-url-kafka: kafka/metrics-monitoring-kafka/README.adoc
+:service-binding-url-kafka: kafka/service-binding-kafka/README.adoc
+
+//OpenShift Service Registry
+:product-long-registry: OpenShift Service Registry
+:product-registry: Service Registry
+:registry: Service Registry
+:product-version-registry: 1
+:service-url-registry: https://console.redhat.com/application-services/service-registry/
+:getting-started-url-registry: registry/getting-started-registry/README.adoc
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:rhoas-cli-getting-started-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:product-long-connectors: OpenShift Connectors
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-browsing-messages"]
+= Browsing messages in the {product-long-kafka} web console
+ifdef::context[:parent-context: {context}]
+:context: browsing-messages
+
+// Purpose statement for the assembly
+[role="_abstract"]
+
+As a developer or administrator, you can use the {product-long-kafka} web console to view and inspect messages for a Kafka topic. You might use this functionality, for example, to verify that a client is producing messages to the expected topic partition, that your topic is storing messages correctly, or that messages have the expected content.
+
+When you select a topic in the console, you can use the *Messages* tab to view a list of messages for that topic. You can filter the list of messages in the following ways:
+
+* Specify a partition and see messages sent to the partition.
+* Specify a partition and offset. See messages sent to the partition from that offset.
+* Specify a partition and a timestamp (that is, date and time) value. See messages sent to the partition from that date and time.
+* Specify a partition and a Unix epoch timestamp value. See messages sent to the partition from that epoch timestamp value.
```

Comment:
I'm not sure I understand the syntax in these bullets, could just be my brain. First is "and see messages" and the others are all "bla. See messages ..."  For each, how about "... and see messages" or "... to see messages" consistently. So "partition and see", "timestamp and see", etc.

Reply:
Yep, good call.